konfera/templates/konfera/base.html

```diff
@@ -11,17 +13,77 @@
     <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
     <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
   <![endif]-->
+  <link rel="stylesheet" href="{% static "konfera/css/pycon.css" %}">
+  <link rel="icon" href="{% static "konfera/images/favicon.ico" %}" sizes="16x16">
+
 </head>
 <body>
-    {% if messages %}
-  <ul class="messages">
-      {% for message in messages %}
-      <li{% if message.tags %} class="{{ message.tags }}"{% endif %}>{{ message }}</li>
-      {% endfor %}
-  </ul>
+
+  <div class="container" id="content" role="main">
+  <!-- content -->
+
+  {% if messages %}
+    {% for message in messages %}
+    <div class="alert alert-success alert-dismissible{% if message.tags %} {{ message.tags }}{% endif %}" role="alert">
+      <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+      {{ message }}
+    </div>
+    {% endfor %}
   {% endif %}
+
   {% block content %}{% endblock %}
+
+  <!-- content end -->
+    <footer class="hidden-print">
+      <div class="row">
+        <div class="col-md-12">
+          <ul class="list-unstyled">
+            <!--
+            TODO: Prepopulate social_media_list from #88
+
+            <li><a href="https://facebook.com/pyconsk">Facebook</a></li>
+            <li><a href="https://twitter.com/pyconsk">Twitter</a></li>
+            <li><a href="https://www.linkedin.com/company/spy-o--z-">LinkedIn</a></li>
+            <li><a href="https://plus.google.com/u/0/b/114745643590416178453/114745643590416178453/posts">Google+</a></li>
+            <li><a href="https://pyconsk.slack.com">Slack</a></li>
+            <li><a href="https://github.com/pyconsk">GitHub</a></li>
+            <li><a href="spy.html">Civic association</a></li>
+            <li><a href="https://groups.google.com/d/forum/pycon-sk"><span class="glyphicon glyphicon-envelope" aria-hidden="true"></span> Mailing list</a></li>
+            <li><a href="mailto:info%40pycon.sk"><span class="glyphicon glyphicon-envelope" aria-hidden="true"></span> info@pycon.sk</a></li>
+            <li class="hide"><a href="../sitemap.xml">sitemap.xml</a></li>
+            -->
+          </ul>
+
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-md-12">
+          <small class="text-center"><a href="https://www.python.org/psf/trademarks/pycon/" target="_blank">PyCon™</a>is a trademark for worldwide conference activities claimed by the <a href="https://www.python.org/psf-landing/" target="_blank">Python Software foundation</a>.</small>
```

Comment:
Let's keep the line length bellow 120 chars in templates as well. This is hard to read.